postgres/9.6/postgresql.conf.sample

```diff
@@ -9,7 +10,7 @@ shared_preload_libraries = 'decoderbufs'
 
 # REPLICATION
 wal_level = logical             # minimal, archive, hot_standby, or logical (change requires restart)
-max_wal_senders = 1             # max number of walsender processes (change requires restart)
+max_wal_senders = 4             # max number of walsender processes (change requires restart)
 #wal_keep_segments = 4          # in logfile segments, 16MB each; 0 disables
 #wal_sender_timeout = 60s       # in milliseconds; 0 disables
-max_replication_slots = 1       # max number of replication slots (change requires restart)
\ No newline at end of file
+max_replication_slots = 4       # max number of replication slots (change requires restart)
```

Comment:
this is really for testing purposes only. In a production system I suppose this can vary according to the various production requirements. The only requirement from a DBZ perspective is that that `max_wal_senders` and `max_replication_slots` >= 1